app/assets/javascripts/discourse/tests/unit/lib/pretty-text-test.js

```diff
@@ -1611,13 +1611,7 @@ var bar = 'bar';
 
   test("watched words replace", function (assert) {
     const opts = {
-      watchedWordsReplace: {
-        "(?:\\W|^)(fun)(?=\\W|$)": {
-          word: "fun",
-          replacement: "times",
-          case_sensitive: false,
-        },
-      },
+      watchedWordsReplace: [["(?:\\W|^)(fun)(?=\\W|$)", false, "fun", "times"]],
```

Comment:
Yeah, 1 line vs 8 lines is nice... but readability suffers, hard to reason what `false` "fun" and "times" are in the new format.

Reply:
It's just a serialization format, mostly used to transfer the data through the ruby<->js boundary. I'd use a binary format if that was an option 😉 (tbh, I would explore viability of e.g. protobuf here 😃)